.travis.yml

```diff
@@ -96,6 +97,8 @@ addons:
       key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
     - sourceline: 'deb http://download.opensuse.org/repositories/home:/delcypher:/z3/xUbuntu_14.04/ /'
       key_url: 'http://download.opensuse.org/repositories/home:delcypher:z3/xUbuntu_14.04/Release.key'
+    - sourceline: 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.7 main'
```

Comment:
The 3.7 repository is already added, so there's some duplication here.

Reply:
Well spotted, fixed.